rerun_py/rerun_sdk/rerun/log/image.py

```diff
@@ -90,6 +102,23 @@ def log_image(
     if interpretable_as_image and num_non_empty_dims != len(shape):
         image = np.squeeze(image)
 
+    if jpeg_quality is not None:
+        # TODO(emilk): encode JPEG in background thread instead
```

Comment:
it might be nice to do this in rust instead in the future to make it more cross-platform and to introduce less python dependencies

Reply:
I agree